src/modules/media/elasticsearch/queryBuilder.ts

```diff
@@ -203,6 +206,17 @@ export class QueryBuilder {
 				);
 				return;
 			}
+			// name and description are multi_match fields, but they are allowed to be queried using is/isNot operator
```

Comment:
Shouldn't this comment be negative?
```suggestion
			// name and description are multi_match fields, but they are not allowed to be queried using is/isNot operator
```

Reply:
name and description are allowed to be queried with is/isNot operator;
query and advancedQuery fields are NOT. clarified the comment.